Code-Hammers/code-hammers
# [CHE-30] Design And Implement Profile Model

### Description

This ticket is part of [CHE-2](https://codehammers.atlassian.net/browse/CHE-2?atlOrigin=eyJpIjoiNTY3YjliODk5NjY2NGJiNzg1NTdiZjk0MDYzYTc1ZjIiLCJwIjoiaiJ9) Create Profile Page story and updates the Profile Model and any code interacting with it.

### Jira Task

[CHE-30](https://codehammers.atlassian.net/browse/CHE-30?atlOrigin=eyJpIjoiMGU1MmZlOTA0YThhNGI2MDkyNmQyYmUyOGJiYjYzZWQiLCJwIjoiaiJ9)

### Testing Instructions

1. `git fetch`
2. `git checkout CHE-30/subtask/Design-And-Implement-profile-Model`
3. `npm run docker-dev` --> verify core functionality is present and navigation to the Profiles Page still displays 
    seeded data.

### Checklist


### Additional Notes, Images, etc.

N/A


[CHE-2]: https://codehammers.atlassian.net/browse/CHE-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHE-30]: https://codehammers.atlassian.net/browse/CHE-30?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
